src/components/footer-indicator/FooterIndicator.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../styles/variables';
+
+.bdl-FooterIndicator {
+    background: $fours;
+    border-radius: 4px 4px 0 0;
+    bottom: 0;
+    height: 32px;
+    margin: 0 15px;
+    position: fixed;
+    width: 170px;
+    z-index: 10;
+}
+
+.bdl-FooterIndicator-content {
+    display: flex;
+    justify-content: flex-start;
+
+    &:focus {
+        outline: none;
```

Comment:
It looks like this element is a div without a tabindex. Is `outline: none` still needed?

Reply:
Yes. The fuzzy blue outline will still show up on focus, which would suggest that this element is clickable when it is not.

Reply:
I see, thanks. It looks like `Tooltip` adds a `tabindex` attribute to its immediate child component.